src/components/auth/login/Login.js

```diff
@@ -3,7 +3,12 @@ import { Link, Redirect } from 'react-router-dom';
 import Form from 'react-bootstrap/Form';
 import { FormInput } from '../../global/FormInput';
 import './Login.css';
+import Button from 'react-bootstrap/Button';
```

Comment:
You can import both Form and Button from bootstrap in one line like `import { Form, Button } from 'react-bootstrap'`.